### PR TITLE
Iss175_mlip-2_mlp_train

### PR DESCRIPTION
**Pull request by JiQi535**
1. Adapt the MTP training part to the latest MLIP-2.
2. Change the default weightings of energy-force-stress for MTP training to 1-0.01-0.001.
3. Further modifications are still needed for MTP MD with MLIP-2.

### CODE REVIEW

File: maml/apps/pes/_mtp.py [hunk 1]
```diff
@@ -543,7 +543,6 @@ def train(
                     "--force-weight={}".format(force_weight),
                     "--stress-weight={}".format(stress_weight),
                     "--init-params=same",
-                    "--auto-min-dist",
```
Comment (JiQi535) [inline maml/apps/pes/_mtp.py:546]: This option is no longer available for mlp train in mlip-2.

File: maml/apps/pes/_mtp.py [hunk 2]
```diff
@@ -105,8 +105,8 @@ def _line_up(self, structure, energy, forces, virial_stress):
             lines.append("{:>24.12f}".format(inputs["Energy"]))
         if "Stress" in inputs:
             format_str = "{:>12s}{:>12s}{:>12s}{:>12s}{:>12s}{:>12s}"
-            format_float = "{:>12f}{:>12f}{:>12f}{:>12f}{:>12f}{:>12f}"
-            lines.append(format_str.format("Stress:  xx", "yy", "zz", "yz", "xz", "xy"))
+            format_float = "{:>16f}{:>12f}{:>12f}{:>12f}{:>12f}{:>12f}"
+            lines.append(format_str.format("PlusStress:  xx", "yy", "zz", "yz", "xz", "xy"))
```
Comment (JiQi535) [inline maml/apps/pes/_mtp.py:109]: The notation of Stress in the .cgf file, which stores training data for MTP, has changed to "PlusStress" in mlip-2.

**Comment by JiQi535** [comment]
1. Adapt the MTP training part to the latest MLIP-2.
2. Change the default weightings of energy-force-stress for MTP training to 1-0.01-0.001.
3. Further modifications are still needed for MTP MD with MLIP-2.

**Comment by chc273** [comment]
@JiQi535 Great work! Could you also check the tests, including lammps tests and evaluation, and make sure that all of them pass? Those tests are skipped on Github actions so they cannot be automatically detected for now. 

Also, an example in notebooks/pes would be extremely helpful. 


Thank you!

**Comment by chc273** [comment]
@JiQi535 @shyuep MTP tests have all been skipped on github actions since the installation of MTP on github actions is not straightforward. @JiQi535 Please run all tests in `maml/apps/tests` to make sure the MTP tests can be passed with the new `mlp`.

**Comment by chc273** [comment]
@JiQi535 Please also follow up on my previous comments regarding adding an example of MTP to `notebooks/pes`. 

One more thing, one thing we can do is also make a backward-compatible API for MTP, i.e., our package should work on both MTP v1 and MTP v2. From your modification, it seems to be reasonable to do so. The work involved seems to be minimal.

**Comment by JiQi535** [comment]
@chc273 The new "mlp train" has passed the test, while I still need to modify the "evaluate" and other functions related to mlip.ini and LAMMPS templates. I have received the missions on (1) an example notebook for MTP and (2) a backward-compatible API for MTP. I will work on them. Thanks!

**Pull request merged**